lib/hexpm/web/views/api/user_view.ex

```diff
@@ -58,12 +58,16 @@ defmodule Hexpm.Web.API.UserView do
     Enum.map(packages, fn package ->
       %{
         name: package.name,
+        repository: repository_name(package),
         url: url_for_package(package),
         html_url: html_url_for_package(package)
       }
     end)
   end
 
+  defp repository_name(%Package{repository_id: 1}), do: "hexpm"
```

Comment:
This clause shouldn't be necessary since repository 1 should have the name "hexpm".